### PR TITLE
Improve the Kubernetes deployment documentation

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR improves the docs for deploying Netdata to a Kubernetes cluster using our Helm chart. It also includes the most up-to-date instructions (updating the default Helm chart configuration) on claiming parent/child nodes to Netdata Cloud.

##### Component Name

area/docs

### CODE REVIEW

File: packaging/installer/methods/kubernetes.md [hunk 1]
```diff
@@ -43,147 +33,138 @@ helm repo add netdata https://netdata.github.io/helmchart/
...
+          memory mode = dbengine
+          dbengine_multihost_disk_space = 4000
+          default memory mode = dbengine
+          
+        [plugins]
+          apps = yes
+          proc = yes
+          go.d = yes
+          python.d = yes
+
```
Comment (ilyam8) [inline packaging/installer/methods/kubernetes.md:86]: why do we want to have data collection plugins enabled on the parent instance?

File: packaging/installer/methods/kubernetes.md [hunk 2]
```diff
@@ -43,147 +33,138 @@ helm repo add netdata https://netdata.github.io/helmchart/
...
+parent:
+  claiming:
+    enabled: true
+    token: "TOKEN"
+    rooms: "ROOM"
+  configs:
+    netdata:
+      data: |
+        [global]
+          memory mode = dbengine
```
Comment (ilyam8) [inline packaging/installer/methods/kubernetes.md:77]: Why dbengine?

File: packaging/installer/methods/kubernetes.md (hunk 1 above)
Comment (joelhans) [reply packaging/installer/methods/kubernetes.md:86]: Good question! I _just_ realized that one of the child pods runs on the same node as the parent, and would have those same plugins enabled. So yes, it's probably best to remove those lines. I'll do that.

File: packaging/installer/methods/kubernetes.md (hunk 2 above)
Comment (joelhans) [reply packaging/installer/methods/kubernetes.md:77]: Now this one I don't have a good answer to. _Why not dbengine?_

Comment (ilyam8) [reply packaging/installer/methods/kubernetes.md:77]: My point is: we are changing the default here. Default is `save`. I'm not 100% sure what stops us from using `dbengine`.

**Review by ilyam8 — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Review by kaskavel — APPROVED** [review]

**Pull request merged**